mmcv/runner/base_runner.py

```diff
@@ -291,10 +291,13 @@ def resume(self,
                resume_optimizer=True,
                map_location='default'):
         if map_location == 'default':
-            device_id = torch.cuda.current_device()
-            checkpoint = self.load_checkpoint(
-                checkpoint,
-                map_location=lambda storage, loc: storage.cuda(device_id))
+            if torch.cuda.is_available():
+                device_id = torch.cuda.current_device()
+                checkpoint = self.load_checkpoint(
+                    checkpoint,
+                    map_location=lambda storage, loc: storage.cuda(device_id))
+            else:
+                checkpoint = self.load_checkpoint(checkpoint)
```

Comment:
if cuda is not available here, we should map weights to cpu?

Reply:
`self.load_checkpoint` has default value